### DISCUSSION TITLE
Add RTMP end Point , only disable or enable the end point

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6199] [Ideas]
Add RTMP end Point , only disable or enable the end point  , no need to remove it completely and to add it again, how to do that
is it possible to add it like restream.io

**Comment by yashtandon113** [comment]
@brixxnatt 
Unfortunately, right now, the endpoint cannot be disabled/enabled. It needs to be removed or added again.

But it would be good to have this feature so I have created the feature request as per your suggestions.

Thanks for sharing your idea. Here is the GitHub feature request issue.
https://github.com/ant-media/Ant-Media-Server/issues/6200
